frontendBuild.sh

```diff
@@ -0,0 +1,10 @@
+source testBuild.sh
+cat interface1.html test/frontend_testing/frontend.html > test/frontend_testing/test.html
+sed -i "" 's|build/spot.js|../../build/spot.js|g' test/frontend_testing/test.html
```

Comment:
try to do this as a single regex, i.e., find '<script src="' and replace with '<script src="../../'